react/features/filmstrip/middleware.web.js

```diff
@@ -278,7 +278,7 @@ MiddlewareRegistry.register(store => next => action => {
         const pinnedParticipants = getPinnedActiveParticipants(state);
         const dominant = getDominantSpeakerParticipant(state);
 
-        if (isTopPanelEnabled(state)) {
+        if (isStageFilmstripTopPanel(state, 2)) {
             const screenshares = state['features/video-layout'].remoteScreenShares;
```

Comment:
Can you add a comment what is 2, as reading there is no hint from the name isStageFilmstripTopPanel.

Reply:
But this is described in the JSDoc of the function.

Reply:
But one needs to go there to read it, which is a distraciton. You don't understand it by reading the PR or just this file. If this was a constant with doc in the file it would also help.

Reply:
Or if it was something checkFilmstripVisibleForNumberOfParticipants(2) or something similar you can understand what is 2.

Reply:
I just pointing it out to make it more readable.

Reply:
In addition the whole story behind this functions that sometimes take number of participants to compare it with the pinned participants and sometimes not is pretty confusing for me. 

@robertpin maybe we should revisit where we need to check for at least 2 pinned participants and where we don't and refactor it.